spec/ddtrace/contrib/delayed_job/delayed_job_active_record.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.configure do |c|
+  c.around do |example|
```

Comment:
By virtue of being `require`d, this is still running around every example in the test suite. This should be changed so it only runs around specific examples when explicitly composed into the example itself (not just by use of `require`.) Similarly, any specs that `require` this file, but do not explicitly compose it into its examples should not perform these migrations.